koku/masu/database/ocp_report_db_accessor.py

```diff
@@ -909,7 +909,16 @@ def populate_pvc_table(self, cluster, pvcs):
         """Get or create an entry in the OCP cluster table."""
         LOG.info(log_json(msg="populating reporting_ocp_pvcs table", schema=self.schema, cluster=cluster))
         for pvc in pvcs:
-            OCPPVC.objects.get_or_create(persistent_volume=pvc[0], persistent_volume_claim=pvc[1], cluster=cluster)
```

Comment:
Would `update_or_create()` work here?

Reply:
It would not! I tested using that method locally before implementing this solution, it impacts the expected behavior because it updates the pvcs associated with the pv. I only want to update the csi_volume_handle.

Reply:
Sounds good. Just wanted to check because that's a new method that was introduced in Django 4.1.